R/data.table.R

```diff
@@ -2771,13 +2771,13 @@ address = function(x) .Call(Caddress, eval(substitute(x), parent.frame()))
 }
 
 # TODO(#6197): Export these.
-J = function(...) {
-  stopf("J() called outside of [.data.table. J() is only intended for use in i.")
-}
+# J = function(...) {
```

Comment:
please add a short comment about why they are commented (what happens when cc is run?)

Reply:
done 👍